parsers/GSO.py

```diff
@@ -52,6 +53,15 @@ def get_api_data(session: Session, url, data):
     return json.loads(session.post(url, json=data).json()["d"])
 
 
+def convert_local_ts_to_datetime(ts: str):
+    # convert local TS e.g. 2024-10-27T00:10:00 to utc
+    return (
+        datetime.strptime(ts, "%Y-%m-%dT%H:%M:%S")
+        .replace(tzinfo=TIMEZONE)
+        .astimezone(timezone.utc)
```

Comment:
This technically don't need to be converted back to UTC. I know the old code did that but it's fine to keep it as the local timezone. It will be converted by the backend anyway.